docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md

```diff
@@ -41,10 +41,31 @@ Following are the [fields](/docs/manage/fields/) which will be created as part o
 
 In this step, you will configure the yaml required for macOS Collection.
 
-1. (Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
-2. Click on the **Download YAML File** button to get the yaml file.
+(Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
 
-<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-YAML.png' alt="YAML" />
+#### Enable process metric collection (Optional)
+
+By default, the collector will not be sending process metrics to Sumo Logic this is because the number of processes running on a host can be very large, this may result in significant increase in Data Points per Minute (DPM).
```

Comment:
```suggestion
By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
```